docs/pages/example/display-and-style-bilingual-labels.html

```diff
@@ -0,0 +1,20 @@
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map', // container id
+    style: 'mapbox://styles/mapbox/streets-v9', // stylesheet location
+    center: [9.49, 49.01], // starting position [lng, lat]
+    zoom: 4 // starting zoom
+});
+
+map.on('load', function(){
+    map.setLayoutProperty('country-label-lg', 'text-field', ['format',
```

Comment:
The example will probably have a more consistent appearance if we set the same (or similar) styling for all three of the country layers? I think it's -lg, -md, and -sm, right?